content/src/roadmaps/license-tasks/apply-for-shop-license.md

```diff
@@ -18,12 +18,11 @@ webflowType: business-license
 webflowId: 5f77298fd2749a78af9cf7df
 licenseCertificationClassification: LICENSE
 summaryDescriptionMd: >
```

Comment:
No >-?

Reply:
Good catch. This is definitely valid (the "-" controls the behavior of multiline strings) and this exists in content that already had `summaryDescriptionMd`, but it's a good opportunity to campground.